### DISCUSSION TITLE
No answer in open tickets

### DISCUSSION THREAD

**Discussion by bearbricknik** [community/community#56916] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it normal that the github support does not answer tickets? Not sure, I have bumped it couple of times throughout the weeks but no answer overall. Does someone experience the same?

**Comment by queenofcorgis** [comment]
Hi @bearbricknik, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by bearbricknik** [comment]
Hello,Yes I understand but unfortunately this case is kind of important cause I can not use my account without this being solved…Am 01.06.2023 um 15:11 schrieb Lili ***@***.***>:﻿
Hi @bearbricknik,
Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.
As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
Thank you!
